h/views/activity.py

```diff
@@ -84,12 +84,41 @@ def group_search(request):
     if request.has_permission('admin', group):
         result['group_edit_url'] = request.route_url('group_edit', pubid=pubid)
 
+    result['more_info'] = 'more_info' in request.params
+
     return result
 
 
 @view_config(route_name='activity.group_search',
              request_method='POST',
              renderer='h:templates/activity/search.html.jinja2',
+             request_param='more_info')
+def group_search_more_info(request):
```

Comment:
Sure. This file is one place where I'm making technical debt that I want to refactor once the rush to get all the features in is over. Grouping all these tightly related views into a Pyramid view class will remove a lot of boilerplate and duplication, and something can probably be done to avoid user - group sidebar duplication as well